golem-ts-agentic/package.json

```diff
@@ -3,7 +3,6 @@
     "private": true,
     "type": "module",
     "scripts": {
-        "preinstall": "pnpm --filter @golemcloud/golem-ts-types-core run build",
```

Comment:
its hard to maintain, though powerful setup.

This is replaced with golem-ts-sdk explicitly pointing to the output of golem-ts-typgen/dist, (rather than dev dependency and installation)